scripts/wsrep_sst_xtrabackup-v2.sh

```diff
@@ -1753,6 +1753,16 @@ if [[ ! -x $XTRABACKUP_24_PATH/bin/$XTRABACKUP_BIN ]]; then
 fi
 XB_2x_VERSION=$($XTRABACKUP_24_PATH/bin/$XTRABACKUP_BIN --version 2>&1 | grep -oe ' [0-9]\.[0-9]\.[0-9]*' | head -n1)
 XB_2x_VERSION=${XB_2x_VERSION# }
+
+if [[ -z "$XB_2x_VERSION" ]]; then
+    wsrep_log_error "******************* FATAL ERROR ********************** "
+    wsrep_log_error "* Cannot determine the xtrabackup 2.x version. "
```

Comment:
I see that this file is not consistent in error log messages.
Please remove the leading asterisk from this line, or add the leading asterisk to all lines.
I suggest the 1st solution (remove).